lib/Command/CirclesMaintenance.php

```diff
@@ -109,7 +109,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		$level = (int)$input->getOption('level');
 
 		if ($reset || $uninstall) {
-			$action = strtolower(($uninstall) ? 'uninstall' : 'reset');
+			$action = $uninstall ? 'uninstall' : 'reset';
```

Comment:
why not keeping the `strtolower()` ?

Reply:
because uninstall and reset are already lower case

Reply:
ok, so now I am wondering why I used `strtolower()` myself ...